src/Core/src/Handlers/Entry/EntryHandler.Android.cs

```diff
@@ -157,7 +157,9 @@ void OnFocusedChange(object? sender, FocusChangeEventArgs e)
 
 		void OnEditorAction(object? sender, EditorActionEventArgs e)
 		{
-			if (e.IsCompletedAction())
+			var currentInputImeFlag = VirtualView?.ReturnType.ToPlatform();
```

Comment:
Do we need another `?` for `ReturnType?.ToPlatform();` ?

Reply:
Is not nullable enum, the default value will be `ReturnType.Default`.